src/collection-preferences/content-display/sortable-item.scss

```diff
@@ -42,9 +42,7 @@ $border-radius: awsui.$border-radius-item;
 .sortable-item-content {
   border-top: awsui.$border-divider-list-width solid awsui.$color-border-divider-default;
   display: flex;
-  flex-wrap: nowrap;
```

Comment:
The component behaves the same without this line, also when the label text becomes long enough to overflow (we have regression tests for this case) —the text wraps as expected and the 3 children are kept in a single row.